docs/developers/devel-env.rst

```diff
@@ -82,6 +82,11 @@ Update the strings with your client ID and secret. After doing this, your
 ListenBrainz development environment is able to authenticate and log in from
 your MusicBrainz login.
 
+.. note:: 
+    Make sure the ``MUSICBRAINZ_CLIENT_ID`` and ``MUSICBRAINZ_CLIENT_SECRET`` parameters are set properly, 
```

Comment:
Please also add the screenshot of the modal linked in the ticket here.

Reply:
done, take a look!